### ISSUE TITLE
SplitPane showing only half of screen

### ISSUE THREAD

**Issue by synga** [ionic-team/ionic#12790]
**Resources:**
Before submitting an issue, please consult our troubleshooting guide (http://ionicframework.com/docs/troubleshooting/) and developer resources (http://ionicframework.com/docs/developer-resources/)

**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

When adding a simple split-pane to a new project sidemenu project the sidemenu and ion-nav component have the same size.

**Expected behavior:**

The sidemenu should have the correct width (28% of screen i guess) and the page should get the remaining space.

**Steps to reproduce:**

**Related code:**

```
<ion-split-pane>
  
  <ion-menu [content]="content">
    <ion-header>
      <ion-toolbar>
        <ion-title>Menu</ion-title>
      </ion-toolbar>
    </ion-header>

    <ion-content>
      <ion-list>
        <button menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
        {{p.title}}
      </button>
      </ion-list>
    </ion-content>
  </ion-menu>

  <!-- Disable swipe-to-go-back because it's poor UX to combine STGB with side menus -->
  <ion-nav [root]="rootPage" #content swipeBackEnabled="false"></ion-nav>
</ion-split-pane>
```

In my app.components just changed the rootPage and the array containing pages

```
  rootPage: any = 'HomePage';

    // used for an example of ngFor and navigation
    this.pages = [
      { title: 'Inicio', component: 'HomePage' },
      { title: 'Clientes', component: 'ClientesPage' },
      { title: 'Pagamentos', component: 'PagamentosPage' },
      { title: 'Parceiros', component: 'ParceirosPage' },
      { title: 'Projetos', component: 'ProjetosPage' },
    ];
```

**Other information:**

Here's a screen shot of the behaviour.

![Current Page](https://user-images.githubusercontent.com/9086829/29942709-358fe750-8e6d-11e7-893f-a81a6b3f7504.png)

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (C:\Users\Wabiz Dese\AppData\Roaming\npm\node_modules)

    @ionic/cli-utils  : 1.9.2
    ionic (Ionic CLI) : 3.9.2

local packages:

    @ionic/app-scripts : 2.1.4
    Ionic Framework    : ionic-angular 3.6.0

System:

    Node : v6.11.1
    npm  : 5.3.0
    OS   : Windows 10
```

**Comment by AmitMY** [comment]
Add `main` attribute to the `ion-nav`

**Comment by kensodemann** [comment]
@synga - please let us know if @AmitMY's suggestion works for you

**Comment by synga** [comment]
So no bug at all, read all the split pane docs but didn't see the main attribute in ion-nav, thanks @AmitMY

**Comment by felix9607** [comment]
worked for me too, thanks...but there is really never a word about this Attribute in your docs sadly.

**Comment by AmitMY** [comment]
I do remember having this issue as well until looking at an example..
@kensodemann Is it realistic to add documentation in the next version, or are docs frozen as well?

**Comment by kensodemann** [comment]
@AmitMY - I have been merging doc related PRs and rolling them into the v4 branch as well, so if you know what you want the docs to say, please submit a PR and one of us will most likely get it merged in for the upcoming 3.7.0 release or the one after that.

**Comment by L96Github** [comment]
Also had this issue, it seems that the `max-width` for `ionic-split-pane` selector is set to `28%`. A temporary fix is to add this to your **`src/app.scss`**:
``` 
ion-split-pane{
    ion-nav{
        max-width: unset !important;
    }
}
```
This way you will`unset` the `max-width` restriction and expand the `ion-nav` selector and keep the side menu the same side.
Hope it helps.
